code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -65,6 +65,23 @@
 	else
 		return ..()
 
+/obj/structure/cult/functional/proc/updatehealth()
+	if(health <= 0)
+		destroy_structure()
+
+/obj/structure/cult/functional/proc/take_damage(damage, damage_type = BRUTE)
+	if(damage_type != STAMINA)
```

Comment:
Is there a cleaner way than doing `if(damage_type == BURN || damage_type == BRUTE`?